src/Sylius/Bundle/CurrencyBundle/Resources/config/doctrine/model/Currency.mongodb.xml

```diff
@@ -23,6 +23,7 @@
         <field fieldName="code" name="code" type="string" length="3" />
         <field fieldName="exchangeRate" name="exchangeRate" type="float" />
         <field fieldName="enabled" name="enabled" type="boolean" />
+        <field name="isBase" column="is-base" type="boolean" />
```

Comment:
Hmmm, every other column is called in camelCase, why changing approach for this one?